hack/run-job-aggragator-analyzor.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+## Run this script to locally analyze data for disruption
```

Comment:
Could you elaborate on this part a little bit, I'm slightly confused on the utility of the fork in logic based on gcs credentials or not.

Reply:
Yeah the intention here is to remove that possible fork, I think the interval of the automation will be like 1-2 week. So if something happened and we wish to update things between that interval, we don't need to manually pull down the data and commit, we can just run this and maintain the consistency in the PRs and datasets in terms of ordering.

Reply:
I'll update the comments for this

Reply:
Added more info for reasoning there